web_01.html

```diff
@@ -0,0 +1,225 @@
+                    </div>
+                    <div class="project__about">
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/typography.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Typography Set</h1>
+                                <p>typography</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/clean_sketch.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Clean Sketch</h1>
+                                <p>creative</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/antic_popcorn.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Antic Popcorn</h1>
+                                <p>illustration</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/website_fonts.jpg"></img>   
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Website Fonts</h1>
+                                <p>typography</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/vectorized_ideas.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
```

Comment:
Always keep the parent class name when using BEM
```suggestion
                                <span class="project__left-border"></span>
                                <span class="project__right-border"></span>
```